server/webhook/merge_request.go

```diff
@@ -115,3 +117,9 @@ func (w *webhook) handleChannelMergeRequest(event *gitlab.MergeEvent) ([]*Handle
 
 	return res, nil
 }
+
+func (w *webhook) sanitizeDescription(description string) string {
```

Comment:
Should this method be in `server/webhook/webhook.go` as it's a general purpose one?

Reply:
sure